README.md

```diff
@@ -50,106 +47,106 @@ make docker-build docker-push IMG=<some-registry>/inferno-autoscaler:tag
+-------------------------------------------------------------------------------------------------------------------------------
+kind-inferno-gpu-cluster-control-plane   nvidia.com/gpu       2          2          NVIDIA-A100-PCIE-40GB          40960     
+kind-inferno-gpu-cluster-worker          amd.com/gpu          2          2          AMD-RX-7800-XT                 16384     
+kind-inferno-gpu-cluster-worker2         intel.com/gpu        2          2          Intel-Arc-A770                 16384     
+-------------------------------------------------------------------------------------------------------------------------------
+```
+
 
 ### To Uninstall
 
 **Delete the APIs(CRDs) from the cluster:**
 
 ```sh
-make uninstall
+make undeploy-inferno-on-kind
 ```
 
-**UnDeploy the controller from the cluster:**
+**Delete the APIs(CRDs) from the cluster:**
 
 ```sh
-make undeploy
+make uninstall
 ```
 
 **Delete cluster**
 
 ```sh
-kind delete cluster -n a100-cluster
+make destroy-kind-cluster
 ```
 
-## Local development
+## Local development vllme setup
 
 Local development will need emulated vllm server, prometheus installed in KinD cluster. 
 
-**Create namespace**
-
+This script already deploys emulated vllm server:
 ```sh
-kubectl create ns monitoring
+make deploy-inferno-emulated-on-kind
 ```
 
-**Install prometheus**
-
-```sh
-helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
-helm repo update
-helm install kube-prometheus-stack prometheus-community/kube-prometheus-stack -n monitoring
-```
-
-**Wait for prometheus installation to complete**
-```sh
-kubectl apply -f samples/local-dev/prometheus-deploy-all-in-one.yaml
-kubectl get -n default prometheus prometheus -w
-kubectl get services
-
-NAME                  TYPE        CLUSTER-IP   EXTERNAL-IP   PORT(S)    AGE
-prometheus-operated   ClusterIP   None         <none>        9090/TCP   17s
-
-```
-
-**Access the server**
+**Expose the promethues server**
 
 ```sh
 kubectl port-forward svc/prometheus-operated 9090:9090
 # server can be accessed at location: http://localhost:9090
 ```
 
-**Create vllm emulated deployment**
+**Check vllm emulated deployment**
 
 ```sh
-kubectl apply -f samples/local-dev/vllme-deployment-with-service-and-servicemon.yaml
-
 kubectl get deployments
 NAME               READY   UP-TO-DATE   AVAILABLE   AGE
 vllme-deployment   1/1     1            1           35s
+```
 
+**Expose the vllme server**
+```sh
 kubectl port-forward svc/vllme-service 30000:80
 ```
 
+**Create variant autoscaling object for controller**
+```sh
+kubectl apply -f hack/vllme/deploy/vllme-setup/vllme-variantautoscaling.yaml
+
+# view status of the variant autoscaling object to get status of optimization
+```
+
 **Load generation**
 
 ```sh
-git clone https://github.com/vishakha-ramani/vllm_emulator.git -b new-metric
-
 #run script
-sh ./loadgen.sh
+cd ./hack/vllme/vllm_emulator
```

Comment:
Could you add more text explaining what this script does? I see a comment on RPM, maybe add that before the command?

Reply:
done